src/main/java/org/openrewrite/maven/AbstractRewriteMojo.java

```diff
@@ -262,15 +267,22 @@ protected LargeSourceSet loadSourceSet(Path repositoryRoot, Environment env, Exe
 
     protected List<Result> runRecipe(Recipe recipe, LargeSourceSet sourceSet, ExecutionContext ctx) {
         getLog().info("Running recipe(s)...");
-        return recipe.run(sourceSet, ctx).getChangeset().getAllResults().stream()
-                .filter(source -> {
-                    // Remove ASTs originating from generated files
-                    if (source.getBefore() != null) {
-                        return !source.getBefore().getMarkers().findFirst(Generated.class).isPresent();
-                    }
-                    return true;
-                })
-                .collect(toList());
+        RecipeRun recipeRun = recipe.run(sourceSet, ctx);
+
+        if (exportDatatables) {
+            String timestamp = LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd_HH-mm-ss-SSS"));
+            Path datatableDirectoryPath = Paths.get("target/rewrite/", "datatables", timestamp);
```

Comment:
Can you use `Paths.get("target", "rewrite", "datatables", timestamp);` here instead? Might be safer for Windows users.

Reply:
Yep, good catch. Corrected.

Reply:
So just to chime in, the Windows `FileSystem` implementation accepts both forward and back slashes without any issues. A *nix based `FileSystem` will only accept forward slashes, so either always using forward slashes, letting the filesystem choose (as illustrated by @ammachado), or using `File.separatorChar` are all valid solutions that work across OS boundaries.